.vsts-dotnet-ci.yml

```diff
@@ -2,6 +2,7 @@ trigger:
 - main
 - exp/*
 - vs*
+- perf/*
```

Comment:
Do we need CI (after commit) for these branches? We'll get the official build, this would just be xplat and running tests--and mostly invisible results.

Reply:
That makes sense for the official build, but what do you need the unsigned CI build for?

Reply:
I think we could remove this for CI build..

Reply:
Updated

Reply:
Oh, I got confused. This is not the official build pipeline yml. Yeah, we do not need it indeed.